notification/src/main/java/com/wfsample/notification/NotificationService.java

```diff
@@ -0,0 +1,35 @@
+package com.wfsample.notification;
+
+import com.wfsample.service.NotificationApi;
+
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+/**
+ * Implementation of Notification Service.
+ *
+ * @author Hao Song (songhao@vmware.com).
+ */
+@Path("/notify")
+@Service
+public class NotificationService implements NotificationApi {
+  AtomicInteger notify = new AtomicInteger(0);
+
+  public Response notify(String trackNum) {
```

Comment:
Can we make the notify API async and return 202 always? and it can error out async later?